rero_ils/modules/documents/serializers/json.py

```diff
@@ -214,3 +215,44 @@ def _process_location_buckets(cls, lib, loc_buckets):
                 # location is filterd by library
                 loc_processed_buckets.append(loc_bucket)
         return loc_processed_buckets
+
+
+class DocumentExportJSONSerializer(JSONSerializer, ABC):
+    """Mixin serializing records as JSON."""
```

Comment:
Can you put more details, in particular why this is different from the default JSON serializer? You can mention this in the commit message.

Reply:
ok, it's done